build.gradle

```diff
@@ -50,13 +56,13 @@ jar {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.1.0')
+    api platform('com.exactpro.th2:bom:4.0.1')
 
-    implementation 'com.exactpro.th2:common:3.37.1'
-    implementation 'com.exactpro.th2:codec:4.7.2'
+    implementation 'com.exactpro.th2:common:3.41.0'
+    implementation 'com.exactpro.th2:codec:4.7.4'
 
-    implementation "org.slf4j:slf4j-log4j12"
-    implementation "org.slf4j:slf4j-api"
+    implementation "org.apache.logging.log4j:log4j-slf4j-impl"
+    implementation "org.apache.logging.log4j:log4j-1.2-api"
```

Comment:
We don't need this here. Please, return back the slf4j-api dependency

Reply:
In common, we moved to this logging implementation because we need support for the first and second versions of log4j. I removed the dependency with the implementation, leaving only the api. Nevertheless, when running tests, logging works as it should.

Reply:
And yes, this versions is fine

Reply:
We don't need the dependency on the log4j-1.2 API. We are using slf4j for logging. log4j is one of the adapter implementations for slf4j. So, we need only dependency on the slf4j API. No dependencies on log4j should be declared in the dependencies block

Reply:
ok, I got the point, thanks